packages/components/Button/src/Button.settings.win32.ts

```diff
@@ -81,6 +81,11 @@ export const settings: IComposeSettings<IButtonType> = [
           borderColor: 'strokeFocus2',
           borderWidth: 2,
         },
+        stack: {
+          style: {
+            height: 22
```

Comment:
Can we change line 49 and 86 to use minHeight instead? I think we may limit a user's ability to change the height of a button if they customize one by using height here.

Reply:
If we use minHeight, than original bug won't be fixed, it will have extra pixels again. I checked, user can override the height
![image](https://user-images.githubusercontent.com/11574680/132377384-219b8a90-7ac1-432b-9e08-875ca69c2f27.png)
![image](https://user-images.githubusercontent.com/11574680/132377627-2f933c01-dba2-4885-9ec9-c14280f1c725.png)

Reply:
What if you were to change the text variant to "heroLargeSemibold" though? Would the height of the stack clip it?

Reply:
This is what I see if "height" is used instead of "minHeight" and I use the variant "heroLargeSemibold":

![image](https://user-images.githubusercontent.com/22876140/132383031-61bc855e-8656-4f12-802e-cca8bcf8e523.png)

whereas before, if I used "heroLargeSemibold" the button would resize to an appropriate height:

![image](https://user-images.githubusercontent.com/22876140/132383133-3012c488-127e-4237-8793-44fe04c6a133.png)

I see how this makes your solution not work anymore though :( maybe we shouldn't merge it if it's going to cause a change like this

Reply:
But to add, I suppose someone could work around this by altering the height of the button when it's focused and unfocused too but that would be a lot of work when it used to work like it's seen in the second picture just fine

Reply:
Yeah breaking the above screenshot would be bad - we should hold off on merging if we can't figure out a way to work around that.